src/commands/languages/README.md

```diff
@@ -0,0 +1,100 @@
+# Languages Command
+
+The `languages` command allows you to manage languages in your Storyblok space. Currently, it supports pulling language configurations from your space.
```

Comment:
```suggestion
The `languages` command allows you to handle internalization features within your Storyblok space. Currently, it supports pulling language configurations from your space.
```

Reply:
Hey @jeremenichelli @alvarosabu, I think both versions are a bit too bold. As it doesn't allow for _managing_ anything at this point, we shouldn't raise any unrealistic expectations. If I'm not mistaken, this currently allows you to retrieve all configured languages as a JSON, similar to the `spaces/me` endpoint, correct?

Reply:
Correct, it only allows pulling the languages available on the space @manuelschroederdev

Reply:
Are there any plans to expand the functionality in the future?

Reply:
@manuelschroederdev not in near future